gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/diagnosticreport/DiagnosticReportMapperTest.java

```diff
@@ -294,6 +295,156 @@ public void When_DiagnosticReportContainsChildObservationComment_Expect_Observat
+                                <translation code="165127005" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" />
+                            </code>
+                            <statusCode code="COMPLETE" />
+                            <effectiveTime>
+                                <center nullFlavor="NI" />
+                            </effectiveTime>
+                            <availabilityTime value="201101111533" />
+                        </ObservationStatement>
+                    </component>
+                    <component typeCode="COMP" contextConductionInd="true">
+                        <NarrativeStatement classCode="OBS" moodCode="EVN">
+                            <id root="E8746411-9D21-4A5C-B70B-9D1FF00D3AE4" />
+                            <text>Option A</text>
+                            <statusCode code="COMPLETE" />
+                            <availabilityTime value="201101111533" />
+                        </NarrativeStatement>
+                    </component>
+                    <component typeCode="COMP" contextConductionInd="true">
+                        <NarrativeStatement classCode="OBS" moodCode="EVN">
+                            <id root="21B95C86-121F-4BC5-A2CD-3E2955F4D604" />
+                            <text>Option C</text>
+                            <statusCode code="COMPLETE" />
+                            <availabilityTime value="201101111533" />
+                        </NarrativeStatement>
+                    </component>
+                    <component typeCode="COMP" contextConductionInd="true">
+                        <ObservationStatement classCode="OBS" moodCode="EVN">
+                            <id root="E0DD7160-64CA-48F3-AB00-B21C39D3FB23" />
+                            <code code="136Q.00" codeSystem="2.16.840.1.113883.2.1.6.2" displayName="Very heavy drinker">
+                                <translation code="228279004" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" />
+                            </code>
+                            <statusCode code="COMPLETE" />
+                            <effectiveTime>
+                                <center nullFlavor="NI" />
+                            </effectiveTime>
+                            <availabilityTime value="201101111533" />
+                        </ObservationStatement>
+                    </component>
+                    <component typeCode="COMP" contextConductionInd="true">
+                        <NarrativeStatement classCode="OBS" moodCode="EVN">
+                            <id root="B141FC3E-4DD2-4412-9655-DF43BBCE290B" />
+                            <text>Some freetext - This is total freetext madness</text>
+                            <statusCode code="COMPLETE" />
+                            <availabilityTime value="201101111533" />
+                        </NarrativeStatement>
+                    </component>
+               </CompoundStatement>
```

Comment:
I see that this is done elsewhere within `DiagnosticReportMapper.java` though would it be better to externalise this and load the resource in?

Reply:
I will see how this can be simplified, maybe I will extract this one as there are more lines of xml compared to other places... from the other hand it is good to have it all in one place so that the test can be quickly read/corrected/modified if needed.